http_server/_test_pipelining.pony

```diff
@@ -1,7 +1,8 @@
 use "net"
-use "time"
 use "pony_test"
 use "random"
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled back